<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>712</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,0,0,0,0,0,1">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Slices</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="sliceSlider">
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>15</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox">
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>15</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="mirrorCheckBox">
          <property name="text">
           <string>Mirror</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="pixelComboBox">
          <item>
           <property name="text">
            <string>Screen Size</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>500x500</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>400x400</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>300x300</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>200x200</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clearWidget">
          <property name="text">
           <string>Clear</string>
          </property>
          <property name="shortcut">
           <string>Ctrl+C</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="grid">
          <property name="text">
           <string>Grid</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="gridSlider">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <widget class="MyQGraphicsView" name="graphicsView">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>97</x>
           <y>31</y>
           <width>500</width>
           <height>500</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>500</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border: inherit;
background-color: rgb(240, 240, 240);
border-radius: inherit;</string>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustIgnored</enum>
         </property>
         <property name="backgroundBrush">
          <brush brushstyle="NoBrush">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,1,0,0,0,0,0">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Line width</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="WidgetDrawLineWidth" name="lineWidthWidget" native="true">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="lineWidthSlider">
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>15</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="singlePainterActivator">
          <property name="text">
           <string>Single Mode</string>
          </property>
          <property name="shortcut">
           <string>Ctrl+M</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="brushButton">
          <property name="maximumSize">
           <size>
            <width>41</width>
            <height>28</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="eraserButton">
          <property name="maximumSize">
           <size>
            <width>41</width>
            <height>28</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="colorButton">
          <property name="text">
           <string>Color</string>
          </property>
          <property name="shortcut">
           <string>Ctrl+Shift+C</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="multiColor">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>718</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew_File"/>
    <addaction name="action_Open_File"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Undo"/>
    <addaction name="action_Redo"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_File"/>
   <addaction name="action_Open_File"/>
   <addaction name="actionSave_As"/>
   <addaction name="action_Undo"/>
   <addaction name="action_Redo"/>
   <addaction name="actionQuit"/>
   <addaction name="action_About"/>
  </widget>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_Undo">
   <property name="text">
    <string>&amp;Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_Redo">
   <property name="text">
    <string>&amp;Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="action_Open_File">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionNew_File">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyQGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>myQGraphicsView.h</header>
  </customwidget>
  <customwidget>
   <class>WidgetDrawLineWidth</class>
   <extends>QWidget</extends>
   <header>widgetDrawLineWidth.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
